java/shims/src/java/org/apache/orc/impl/HadoopShims.java

```diff
@@ -35,6 +35,7 @@ enum DirectCompressionType {
     ZLIB_NOHEADER,
     ZLIB,
     SNAPPY,
+    BROTLI,
```

Comment:
IIUC, shims here are used to workaround codec provided by Hadoop libraries of different versions. It seems that we do not need this line.

Reply:
Thanks I will remove it.